testing_and_setup/compass/ocean/cosine_bell/RMSE/default/compute_rmse.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import xarray as xr
+import matplotlib
+import matplotlib.pyplot as plt
+import os 
+
+%matplotlib inline
+matplotlib.rcParams['figure.figsize'] = (16, 16) # Large figures
+dpi=200;
+
+### axis_font = {'fontname':'Arial', 'size':'18'}    
+title_font = {'fontname':'Arial', 'size':'32', 'color':'black', 'weight':'normal'}
+matplotlib.rc('xtick', labelsize=28)
+matplotlib.rc('ytick', labelsize=28)
+
+def rmse(resTag, sliceTime):
+# resTag is the resolution to compute RMSE
+# sliceTag is the time slice, assumes 1 hour output and same for all cases
+    fid = open(resTag+'/default/init_step/namelist.ocean','r')
+    temp = fid.readlines()
+
+    fid.close()
+    for i, line in enumerate(temp):
+        if "config_cosine_bell_lat_center" in line: 
+            ii = line.find('=')+1
+            latCent = float(line[ii:])
+        if "config_cosine_bell_lon_center" in line: 
+            ii = line.find('=')+1
+            lonCent = float(line[ii:])
+        if "config_cosine_bell_radius" in line: 
+            ii = line.find('=')+1
+            radius = float(line[ii:])
+        if "config_cosine_bell_psi0" in line: 
+            ii = line.find('=')+1
+            psi0 = float(line[ii:])
+        if "config_cosine_bell_vel_pd" in line: 
+            ii = line.find('=')+1
+            pd = float(line[ii:])
+
+    init = xr.open_dataset('../../'+resTag+'/default/init_step/initial_state.nc')
```

Comment:
I really prefer people alter strings with `.format()` instead of string "addition".  This may be may own prejudice but I don't see string addition used by seasoned python programmers.
```suggestion
    init = xr.open_dataset('../../{}/default/init_step/initial_state.nc'.format(resTag))
```